Typo fixes

luzpaz opened the pull request:
+ User facing typos
+ trivial source comment typos
+ superflous double whitespace issues
Found using `codespell -q 3 --skip="./build/wix/Localization,./res/translations,./lib" -I ../mixxx-whitelist.txt` where the contents of the whitelist were:
```
iff
freee
substract
```

daschuer commented:
Thanks!
Uhhh, why are this more than the original one? Much more merge conflict risk. :-/

luzpaz commented:
@daschuer I wanted to introduce changes in several PRs but since `lib/` was irrelevant I thought I'd just push them all.

daschuer commented:
It would be cool to have an PR of all non conflicting changes. 
But it is a crazy work to find it out. You need to merge all important pending PRs and revert the conflicting changes.
Since the expected conflicts are easy probably to resolve, merging this just before beta seems to be the best option.

Be-ing commented:
I am planning on doing a thorough review of user visible strings before the beta.

uklotzde commented:
We still have major feature PRs pending for the 2.1 beta. I suggest to defer merging these minor changes at least until the beta has been released.

In this special case I even recommend rebasing for resolving merge conflicts instead of merging to prevent any kind unwanted side-effects. It is easy to review the changes if only comments and formatting are affected.

uklotzde commented:
Any user facing typos in translatable strings should go into a separate PR that needs to be merged sooner before releasing the beta. I didn't find any in this PR by quickly browsing through it.

luzpaz commented on res/controllers/Pioneer-CDJ-HID.js:
```diff
@@ -189,7 +189,7 @@ function PioneerCDJController() {
 
     // Pitch on CDJ sends -1000 to 1000, reset at 0, swap direction
-    this.pitchScaler = function(gruop,name,value) { return -(value/1000); }
+    this.pitchScaler = function(group,name,value) { return -(value/1000); }
```
this looks important

luzpaz commented on res/controllers/Vestax Typhoon.midi.xml:
```diff
@@ -3,7 +3,7 @@
     <name>Vestax Typhoon</name>
     <author>bestdani</author>
-    <description>bestdani's controller capping for Vestax Typhoon in Mixxx 2.0+ and script. The controller reacts as descriped on the console. Some elements are taken from Bill Good's original mapping.</description>
+    <description>bestdani's controller capping for Vestax Typhoon in Mixxx 2.0+ and script. The controller reacts as described on the console. Some elements are taken from Bill Good's original mapping.</description>
```
user-facing

uklotzde commented:
Thank you for your understanding 👍 I think some of us are currently working night and day to make this beta actually happen ;)

Commit:
Typo fixes 

+ User facing typos
+ trivial source comment typos
+ superflous double whitespace issues
+ found redundant comment typo
Found using `codespell -q 3 --skip="./build/wix/Localization,./res/translations,./lib" -I ../mixxx-whitelist.txt` where the contents of the whitelist were:
```
iff
freee
substract
```

luzpaz commented:
Honestly, most of these are pretty much trivial. You can probably wait until after the 2.11 release

luzpaz committed:
More misc. typos

luzpaz committed:
follow-up typo

luzpaz commented on src/waveform/renderers/qtwaveformrenderersimplesignal.cpp:
```diff
@@ -205,7 +205,7 @@ void QtWaveformRendererSimpleSignal::draw(QPainter* painter, QPaintEvent* /*even
     }
 
-    //If channel are not displayed separatly we nne to close the loop properly
+    //If channel are not displayed separately we need to close the loop properly
```
`nne` is meant to be `need`, right?

luzpaz committed:
More follow-up typos

Please review since there are some source changes

daschuer commented:
Before merge, we need your permission. 
Please sign https://docs.google.com/a/mixxx.org/spreadsheet/viewform?formkey=dEpYN2NkVEFnWWQzbkFfM0ZYYUZ5X2c6MQ
and comment here when done.

luzpaz commented:
@daschuer signed. Green light

daschuer commented:
Thank you. LGTM

daschuer commented:
I like to add you name to the Contributor List in Mixxx's about box. Most of us are listed with their real names, should I use "luzpaz" instead?

daschuer commented:
Ping!

luzpaz commented:
@daschuer yea.. `luzpaz` is fine. thanks for merging.

daschuer commented:
Done.

daschuer commented:
Hi @luzpaz, please contact me regarding a contributor gift. Thank you.

luzpaz commented:
@daschuer  how do you prefer I contact you?

daschuer commented:
via daschuer at mixxx org
